src/store_engine/store_hooks.ts

```diff
@@ -15,8 +15,8 @@ export function useStoreProvider() {
     __spreadsheet_stores__: container,
     getStore: <T extends StoreConstructor>(Store: T) => {
       const store = container.get(Store);
-      const mutators: readonly string[] = store.mutators;
-      return proxifyToRender(env, store, mutators);
+      // TODO check this, it shouldn't be/use a hook here
```

Comment:
why not?

Reply:
because `useStoreRenderProxy` uses `useCompent` which is only available in the `setup` of components. (right click on a cell, Insert link => boom)

Reply:
ah right :'(